src/tictacclojure/board.clj

```diff
@@ -28,9 +28,14 @@
     (partial replace-position-with-mark mark position)
     board))
 
-(defn moves-available
-  [board-with-move]
-  (filter integer? board-with-move))
+(defn positions-available
```

Comment:
A correction that I just had myself: available-positions seems to be more correct :)